src/System.Drawing.Common/System.Drawing.Common.sln

```diff
@@ -30,8 +30,8 @@ Global
 		{4B93E684-0630-45F4-8F63-6C7788C9892F}.Debug|Any CPU.Build.0 = netcoreapp-Debug|Any CPU
 		{4B93E684-0630-45F4-8F63-6C7788C9892F}.Release|Any CPU.ActiveCfg = netcoreapp-Release|Any CPU
 		{4B93E684-0630-45F4-8F63-6C7788C9892F}.Release|Any CPU.Build.0 = netcoreapp-Release|Any CPU
-		{191B3618-FECD-4ABD-9D6B-5AC90DC33621}.Debug|Any CPU.ActiveCfg = netcoreapp2.0-Unix-Debug|Any CPU
-		{191B3618-FECD-4ABD-9D6B-5AC90DC33621}.Debug|Any CPU.Build.0 = netcoreapp2.0-Unix-Debug|Any CPU
+		{191B3618-FECD-4ABD-9D6B-5AC90DC33621}.Debug|Any CPU.ActiveCfg = netcoreapp2.0-Windows_NT-Debug|Any CPU
```

Comment:
Out of curiosity, was this intentional?

Reply:
Ehh, this is because I was switching between configurations in Visual Studio... I'll keep this since Windows_NT should be the default. I'll just have to remember not to re-commit it in the Unix state.